src/styles/Form.module.css

```diff
@@ -0,0 +1,68 @@
+
+.formmaincontainer {
+  display: flex;
+  justify-content: center;
+  background-color: #e8e8dc;
```

Comment:
Please stick to the design and use the same color for the background color. Zeplin gives all the necessary CSS values, kindly use them.
<img width="1345" alt="Screenshot 2023-03-18 at 2 21 35 PM" src="https://user-images.githubusercontent.com/25887253/226102280-5e712a4c-9092-427b-b6c8-d97b5348620f.png">
<img width="1211" alt="Screenshot 2023-03-18 at 2 23 55 PM" src="https://user-images.githubusercontent.com/25887253/226102283-375c2c08-f9ca-41b3-a766-7b26e60dc775.png">